dandi/files/bases.py

```diff
@@ -469,6 +480,21 @@ def get_validation_errors(
         else:
             # make sure that we have some basic metadata fields we require
             try:
+                # Ensure latest version of NWB Inspector is installed and used client-side
+                current_version, max_version = self._get_inspector_versions()
+                if current_version < max_version:
+                    errors.extend(
```

Comment:
I would not make demands such strong as to make it The only error and overall error out.

Please 
- add a dedicated try/except around the point of your interaction with the Internet. I might be offline and I still want to be able to run validation!
- make it into a `lgr.warning` and proceed "as usual" without `return`.

Reply:
Our of curiosity, how does `dandi upload` handle internet connectivity? Would it error out here or somewhere below:

https://github.com/dandi/dandi-cli/blob/ef097cc124629556d94544b4ca40f2f1ee567425/dandi/upload.py#L67-L69

Reply:
Fixed as requested, though.

Reply:
> out here or somewhere below:

I bet it should fail somewhere "here"  -- just try? ;)